apiserver/application/conversions.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2015 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package application
+
+import (
+	"github.com/juju/errors"
+
+	"github.com/juju/juju/apiserver/params"
+	"github.com/juju/juju/environs"
+	"github.com/juju/juju/network"
+	"github.com/juju/juju/state"
+)
+
+// ParamsFromProviderSpaceInfo converts a ProviderSpaceInfo into the
+// equivalent params.RemoteSpace.
+func ParamsFromProviderSpaceInfo(info *environs.ProviderSpaceInfo) params.RemoteSpace {
```

Comment:
No, they don't, good call - holdover from when they were in `apiserver/common/crossmodelcommon`.